examples/JAX/train.py

```diff
@@ -0,0 +1,477 @@
+# Copyright 2023 The Flax Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""ImageNet example.
+
+This script trains a ResNet-50 on the ImageNet dataset.
+The data is loaded using tensorflow_datasets.
```

Comment:
Does it automatically download ImageNet? That'll take forever. There must be some argument that allows users to specify a path to a pre-downloaded copy of ImageNet.